include/experimental/__p0009_bits/compressed_pair.hpp

```diff
@@ -40,19 +40,19 @@ template <class _T1, class _T2, class _Enable = void> struct __compressed_pair {
   }
 
   MDSPAN_INLINE_FUNCTION_DEFAULTED
-  constexpr __compressed_pair() noexcept = default;
+  constexpr __compressed_pair() = default;
```

Comment:
noexcept ness for defaulted functions should be defaulted, required to have the expected behavior for the corresponding mdspan operations.

Reply:
Shouldn't `noexcept`-ness just be conditional on what the members do?

Reply:
That is what implicitly happens for defaulted functions.